### PR TITLE
Pairtools recipe

### PR DESCRIPTION
**Pull request by golobor**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).

Hi @bioconda/core,
with this pull request I added a new recipe for [pairtools](https://github.com/mirnylab/pairtools), a set of command-line tools for [Hi-C](https://en.wikipedia.org/wiki/Chromosome_conformation_capture#Hi-C_(all-vs-all)) data processing. 

This is my first commit to bioconda, and I would appreciate your help in reviewing it! This recipe passes the CircleCI tests.

Thanks!
Anton.

### CODE REVIEW

File: recipes/pairtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 172c1bc9a043f4ecaf2b464bddd7f17dcc3e0fe417936909062485ab107c23a2
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/pairtools/meta.yaml]: please start with 0

File: recipes/pairtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    - pairtools = pairtools:cli
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - cython
+    - numpy
```
Comment (bgruening) [inline recipes/pairtools/meta.yaml]: If you are compiling against numpy you should use pinned_compatible() in your runtime.

File: recipes/pairtools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    - pairtools --help
+
+about:
+  home: https://github.com/mirnylab/pairtools
+  license: MIT
+  license_family: MIT
+  summary: 'CLI tools to process mapped Hi-C data'
+  license_file: LICENSE
+  doc_url: http://pairtools.readthedocs.io/
+  dev_url: https://github.com/simplejson/simplejson
```
Comment (bgruening) [inline recipes/pairtools/meta.yaml]: This looks wrong.

**Comment by golobor** [comment]
@bgruening , thank you very much for this review, it was very helpful! The last issue was quite embarrassing... :) I fixed these three issues and committed the changes; the updated recipe passes the tests.

**Comment by bgruening** [comment]
@golobor looks great! Thanks a lot!

**Comment by golobor** [comment]
@BiocondaBot autobump pairtools

**Pull request merged**